RecoBTag/DeepFlavour/python/DeepFlavour_cff.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoBTag.DeepFlavour.DeepFlavourTagInfos_cfi import pfDeepFlavourTagInfos
+from RecoBTag.DeepFlavour.DeepFlavourJetTags_cfi import pfDeepFlavourJetTags
+from CommonTools.PileupAlgos.Puppi_cff import puppi
+from PhysicsTools.PatAlgos.slimming.primaryVertexAssociation_cfi import primaryVertexAssociation
```

Comment:
this is technically not a legal dependence : modules running in RECO step should not depend on PhysicsTools.PatAlgos. In this case the issue is minor: the primaryVertexAssociation_cfi can be simply moved to CommonTools/RecoAlgos/python